src/nvim/option.c

```diff
@@ -1581,13 +1581,12 @@ static vimoption_T
   {"ttyfast",  "tf",  P_BOOL|P_NO_MKRC|P_VI_DEF,
    (char_u *)&p_force_on, PV_NONE,
    {(char_u *)TRUE, (char_u *)0L} SCRIPTID_INIT},
+
+   // NO-OP
   {"ttymouse",    "ttym", P_STRING|P_NODEFAULT|P_NO_MKRC|P_VI_DEF,
```

Comment:
we should probably remove it, else we will get more tickets asking why ttymouse works in Vim but not Neovim. Already fielding multiple such issues for `t_Co` and friends.